python/adelphi/tests/integration/resources/nb-base-schema.cql

```diff
@@ -0,0 +1,24 @@
+CREATE KEYSPACE testkeyspace WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'}  AND durable_writes = true;
```

Comment:
Can't we use the same base schema as the anonymization tests? We can probably ignore the unsupported data types for now (suppress them from the nb output).

Reply:
At the moment unsupported types break everything.  There are plans to fix this (see https://github.com/datastax/adelphi/issues/107 for additional details) but this has to be done carefully.  At minimum you have to make sure any primary keys are supported types (for reasons discussed in that ticket).  That ticket also envisions extending base type support to ints and (perhaps) date/time vals which should cover a lot of schemas... which I think includes the base schema as well?

I could see maybe revisiting this after https://github.com/datastax/adelphi/issues/107 is completed.  I really don't love the idea of maintaining a collection of static schema files for individual tests... but at the same time if that's the most efficient way to do it I don't want to close off that avenue.

So in short I guess I'm agreeing with you, but I'm not sure we have enough infrastructure to really do it right yet.  Fortunately I was planning on picking up https://github.com/datastax/adelphi/issues/107 as the next ticket... so maybe I'll take another look at this as part of that work.  Seem reasonable?